formcentral/Dockerfile

```diff
@@ -1,20 +1,16 @@
-FROM golang:alpine AS build
+FROM golang:alpine
+
 RUN apk add --no-cache \
-        gcc \
-        libc-dev
+    bash \
+    gcc \
+    libc-dev
+
 RUN mkdir /app
 WORKDIR /app
 
-ADD go.mod .
```

Comment:
@nandanrao, according to Dockerfile best practices, https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#add-or-copy, it is advised to use `COPY` in favor of `ADD`.

Reply:
Good catch!